lib/translator/BUILD

```diff
@@ -101,3 +102,16 @@ cc_library(
   ],
   visibility = ["//visibility:public"],
 )
+
+cc_library(
+  name = "write_lib",
+  hdrs = ["write.h"],
+  srcs = ["write.cc"],
+  deps = [
+    ":common",
+    "//lib:scsi_lib",
```

Comment:
`lib:scsi_lib` doesn't need to be included because `:common` already has it as a dependency. Also, we are no longer using absl::Span so i think the absl/base dependency can be removed

Reply:
done!